packages/in_app_purchase/in_app_purchase_android/lib/src/billing_client_wrappers/sku_details_wrapper.dart

```diff
@@ -5,7 +5,7 @@
 import 'package:flutter/foundation.dart';
 import 'package:json_annotation/json_annotation.dart';
 
-import 'billing_client_wrapper.dart';
+import '../../billing_client_wrappers.dart';
```

Comment:
Prefer directly importing billing_client_manager.dart over the export file which contains extraneous imports.